view-user/src/components/common/ReachCount/styles.module.css

```
@@ -0,0 +1,42 @@
+.reachCountContainer {
+  width: 40vw;
+  height: 17.75vw;
+  background-color: #ff3342;
+  border-radius: 2.85vw;
+  display: flex;
+  position: relative;
```

```suggestion
  position: relative;
  font-family: "Noto Sans JP";
```
font-familyはcontainerに当てちゃってもいいかもですね